plugin/src/main/java/net/dzikoysk/funnyguilds/data/DataModel.java

```diff
@@ -61,7 +61,7 @@ default void validateLoadedData() {
     }
 
     static DataModel create(FunnyGuilds plugin, PluginConfiguration.DataModel modelType) {
-        if (modelType == PluginConfiguration.DataModel.MYSQL) {
+        if (modelType == PluginConfiguration.DataModel.MYSQL || modelType == PluginConfiguration.DataModel.MARIADB) {
```

Comment:
Może po prostu dla FLAT zrobić ifa, a jak nie wejdzie do niego to SqlDataModel wtedy? Na razie i tak mamy same SQLe poza flatem